crates/common_enums/src/enums.rs

```diff
@@ -1101,6 +1101,7 @@ pub enum PaymentExperience {
 #[strum(serialize_all = "snake_case")]
 pub enum PaymentMethodType {
     Ach,
+    AchBankTransfer,
```

Comment:
Why is this separate variant required for bank transfers?

Reply:
This is required since a given `payment_method_type` can map to a single `payment_method`. For payout use-cases, the `payment_method` for bank payouts is `BankTransfer`.

Ach, Bacs and Sepa are mapped to `BankDebit`. So we need new variants

Reply:
@kashif-m The `PaymentMethodType` enum is shared by a lot of modules including Payments, Smart Routing, and Payouts now. So we shouldn't make concessions for any particular module if it's not going to be useful for the other dependents.

It's not necessary for a `PaymentMethodType` to map onto a single payment method. Certain payment method types like `Ach` like you mentioned can be mapped onto multiple payment methods. Hence wherever we are dependent on payment method type information in context of payment method information, we need to have both a `PaymentMethod` as well as a `PaymentMethodType`.

If you have come across a case where we're breaking this convention, then that's a discrepancy. Please create an issue for it and inform me.

Reply:
@vspecky you can take a look at this function `validate` which uses a `foreign_from`. This makes sure a given variant in `PaymentMethodType` can only map to a single variant `PaymentMethod`.

ref - https://github.com/juspay/hyperswitch/blob/main/crates/router/src/types/api/payment_methods.rs#L24